introduction_to_amazon_algorithms/xgboost_abalone/xgboost_abalone_dist_script_mode.ipynb

```diff
@@ -47,7 +47,6 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# ensure sagemaker version >= 1.35.0\n",
```

Comment:
Would it be helpful to include a comment saying, "ensure sagemaker version >= 1.56.3"?